compiler/src/iree/compiler/Codegen/Utils/Utils.h

```diff
@@ -308,8 +311,17 @@ struct VectorizationTileSizes {
 /// chain.
 std::optional<VectorizationTileSizes> inferSizesFromIR(Value val);
 
-/// Returns the result sizes and vector input sizes of the linalg.unpack op. The
-/// inferred bounding size is returned if it is dynamic shape. Returns
+/// Returns the inferred input-vector-sizes for the `op`, given the inferred
+/// vector sizes for the write operation. Returns std::nullopt, if it fails to
+/// compute the sizes.
+/// For now, it only supports non-scalable vectors.
+std::optional<SizesAndScalableFlags>
+getVectorInputSizesFromDestTiles(linalg::UnPackOp op,
+                                 ArrayRef<int64_t> vectorSizes,
```

Comment:
This confused me in the past, hence the suggestion.
```suggestion
                                 ArrayRef<int64_t> writeVectorSizes,
```